.github/workflows/sitemap_generator.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Sitemap for all the domains and Upload to R2
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_domain:
+        description: "Base domain name"
+        default: "deriv.com"
+        required: true
+        type: string
+
+jobs:
+  update-sitemap:
+    runs-on: ubuntu-latest
+    env:
+        BASE_DOMAIN: ${{ github.event.inputs.base_domain }}
+    steps:
+
+      - name: Download sitemap.xml
+        run: curl -O https://$BASE_DOMAIN/sitemap.xml
```

Comment:
@ali-hosseini-deriv how initial sitemap is generated?
Do you fetch it from webflow?

Reply:
yes, webflow generates the original one automatically, we fetch it and create one copy for each domain, replace the domain name and then publish to R2 bucket.

Reply:
So, is `https://$BASE_DOMAIN/sitemap.xml` that is effectively `https://deriv.com//sitemap.xml` a Webflow resource?

Reply:
that's right @dan-deriv 
I made it dynamic because right now its deriv.me,
But not sure if in the future it changes to deriv.com or ...

Reply:
But as this is not a webflow address, do you rely on Cloudflare, that will proxy-pass the request to Webflow?

Reply:
This workflow will be triggered manually, 
And whoever triggers this should provide the correct domain/url.
base_domain is coming from the inputs which is filled in the dispatch modal of github.
The one that I added as default is for the time that we go live on deriv.com. I considered the deriv.com as the main, anyways that also can be changed.

Reply:
If we put a redirect rule on this, won't it redirect you to the R2 version? :-)

Reply:
@chrisgodfrey-deriv
We should not redirect the main domain, as an example deriv.com because it contains the correct urls in the sitemap file.
Just need to make sure about definition of the main domain in the webflow. we need to make sure that It stays the same all the time.
Anyways I also added it as an input here. so we can provide the URL manually if needed.